RevenueCatUI/Helpers/Variables.swift

```diff
@@ -19,45 +20,50 @@ protocol VariableDataProvider {
 
 }
 
+struct VariableMatch {
+    let variable: String
+    let range: Range<String.Index>
+}
+
 /// Processes strings, replacing `{{variable}}` with their associated content.
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 enum VariableHandler {
 
     static func processVariables(
         in string: String,
         with provider: VariableDataProvider
     ) -> String {
+        let matches = extractVariables(from: string)
```

Comment:
Nit per our style guide:
```suggestion
        let matches = Self.extractVariables(from: string)
```

Reply:
my bad! Updated